Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -123,7 +123,8 @@ def search_query(self, body: Dict[str, Any]) -> Dict[str, Any]:
                                       full_url=f'{self._base_url}attributes/restSearch',
                                       resp_type='json',
                                       headers=headers,
-                                      data=json.dumps(body))
+                                      data=json.dumps(body),
+                                      timeout=int(demisto.get(demisto.params(), 'timeout')))
```

Comment:
Let's change it and set the `timeout` parameter as a class attribute and set it in the init of the client.

Reply:
Fixed